### PR TITLE
BUG: Mark translatable strings in DataProbe module

### PR DESCRIPTION
**Pull request by mhdiop**
See Slicer/SlicerLanguagePacks#21

### CODE REVIEW

File: Modules/Scripted/DataProbe/DataProbe.py [hunk 1]
```diff
@@ -288,9 +290,9 @@ def _roundInt(value):
         if hasattr(self.frame.parent(), 'text'):
             sceneName = slicer.mrmlScene.GetURL()
             if sceneName != "":
-                self.frame.parent().text = "Data Probe: %s" % self.fitName(sceneName, nameSize=2 * self.nameSize)
+                self.frame.parent().text = _("Data Probe: %s") % self.fitName(sceneName, nameSize=2 * self.nameSize)
```
Comment (lassoan) [inline Modules/Scripted/DataProbe/DataProbe.py:293]:
```suggestion
                self.frame.parent().text = _("Data Probe: {sceneName}").format(sceneName=self.fitName(sceneName, nameSize=2 * self.nameSize))
```

**Review by lassoan — COMMENTED** [review]
Thank you it mostly looks good. Please review/fix the inline comments.

File: Modules/Scripted/SegmentEditor/SubjectHierarchyPlugins/SegmentEditorSubjectHierarchyPlugin.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def onSegment(self):
             segmentationNode = slicer.mrmlScene.AddNewNodeByClass('vtkMRMLSegmentationNode')
             editorWidget.parameterSetNode.SetAndObserveSegmentationNode(segmentationNode)
         # Name segmentation node based on the volume
-        segmentationNode.SetName(volumeNode.GetName() + _('_Segmentation'))
+        segmentationNode.SetName(volumeNode.GetName() + '_Segmentation')
```
[the remark this answers is not in the document]
Comment (mhdiop) [reply Modules/Scripted/SegmentEditor/SubjectHierarchyPlugins/SegmentEditorSubjectHierarchyPlugin.py:106]: Ok, I didn't made the volume node's name translatable. I'll then take that into account in the next commits.

Thanks.

**Review by lassoan — APPROVED** [review]
Thank you it looks good!

**Pull request merged**